doc/introduction/remote.rst

```diff
@@ -352,52 +296,101 @@ Rgate(1.902) | (q[0])
-        ops.S2gate(1.0) | (q[3], q[7])
+    complex array A1[4, 4] =
+        -0.13879438-0.47517904j, -0.29303954-0.47264099j, -0.43951987+0.12977568j, -0.03496718-0.48418713j
+        0.06065372-0.11292765j, 0.54733962+0.1215551j, -0.50721513+0.56195975j, -0.15923161+0.26606674j
+        0.42212573-0.53182417j, -0.2642572+0.50625182j, 0.19448705+0.28321781j, 0.30281396-0.05582391j
+        0.43097587-0.30288974j, 0.07419772-0.21155126j, 0.28335618-0.13633175j, -0.75113453+0.09580304j
+
+    # Initial states are two-mode squeezed states
+    S2gate(1.0, 0.0) | [0, 4]
+    S2gate(1.0, 0.0) | [1, 5]
+    S2gate(1.0, 0.0) | [3, 7]
 
-        unitary() | q[:4]
-        unitary() | q[4:]
+    # Apply the unitary matrix above to
+    # the first pair of modes, as well
+    # as a beamsplitter
+    Interferometer(A0) | [0, 1, 2, 3]
+    BSgate(0.543, 0.123) | [2, 0]
+    Rgate(0.453) | 1
+    MZgate(0.65, -0.54) | [2, 3]
+
+    # Duplicate the above unitary for
+    # the second pair of modes
+    Interferometer(A1) | [4, 5, 6, 7]
+    BSgate(0.543, 0.123) | [6, 4]
+    Rgate(0.453) | 5
+    MZgate(0.65, -0.54) | [6, 7]
+
+    # Perform a PNR measurement in the Fock basis
+    MeasureFock() | [0, 1, 2, 3, 4, 5, 6, 7]
+
+After you have created your Blackbird script, either by exporting it from
+Strawberry Fields or writing it by hand, it can be remotely executed using the command line,
+
+.. code-block:: console
+
+    $ sf run program1.xbb --output out.txt
+
+After executing the above command, the result will be stored in ``out.txt`` in the
+current working directory. You can also omit the ``--output`` parameter to print the
+result to the screen.
+
+Furthermore, saved Blackbird scripts can be imported as a Strawberry Fields program
+using the :func:`~.load` function:
+
+>>> prog = load("test.xbb")
 
 
 Embedding bipartite graphs
 --------------------------
 
-We can embed bipartite graphs, with the restriction that the singular
-values form the set :math:`\{0, d\}` for some real value :math:`d`.
+The currently available hardware devices support embedding bipartite graphs,
+i.e., those with adjacency matrices
 
-The matrix :math:`B` represents the edges between the two sets of
-vertices in the graph, and :math:`A` is the full adjacency matrix
-:math:`A = \begin{bmatrix}0 & B\\ B^T & 0\end{bmatrix}`. Here, we will
-consider a complete bipartite graph, since we know that the singular
-values are of the form :math:`\{d, 0\}`.
+.. math:: A = \begin{bmatrix}0 & B\\ B^T & 0\end{bmatrix}
+
+where :math:`B` represents the edges between the two sets of
+vertices in the graph. However, the devices are currently restricted
+to bipartite graphs with equally sized partitions, such that the singular values form the set :math:`\{0, d\}`
+for some real value :math:`d`.
+
+Here, we will
+consider a `complete bipartite graph <https://en.wikipedia.org/wiki/Complete_bipartite_graph>`_,
+since the singular values are of the form :math:`\{0, d\}`.
```

Comment:
Didn't like my `zeros_like` suggestion? :crying_cat_face:

Reply:
Oh, I wasn't sure what you meant by it! And then in the process of making the new PR, accidentally lost track of that comment. Did you mean write `A` using `zeros_like`?

Reply:
Instead of using `0*B`, use `np.zeros_like(B)`